build/travis_build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -ex
+
+root_dir="$(cd "$(dirname $0)/.." && pwd -P)"
+
+"$root_dir/build/configure"
+
+if [[ "$(uname)" == Linux ]]; then
+    clang_revision=268813-1
```

Comment:
Recent Ubuntu versions available in Travis seem to have too old clang, so here we simply grab prebuilt binaries which are used to build Chromium.